mix.exs

```diff
@@ -41,7 +41,8 @@ defmodule ExDoc.Mixfile do
     [
       clean: [&clean_test_fixtures/1, "clean"],
       setup: ["deps.get", "cmd npm install --prefix assets"],
-      build: ["cmd npm run --prefix assets build", "compile --force", "docs"]
+      build: ["cmd npm run --prefix assets build", "compile --force", "docs"],
+      lint: ["format", "cmd npm run --prefix assets/ lint"]
```

Comment:
I am not too sure if we want to format the Elixir code given we may be running different versions of Elixir to submit the PRs